pennylane_lightning_kokkos/src/simulator/MeasuresKokkos.cpp

```diff
@@ -0,0 +1,5 @@
+#include "MeasuresKokkos.hpp"
+
+// explicit instantiation
+template class Pennylane::Simulators::MeasuresKokkos<float>;
+template class Pennylane::Simulators::MeasuresKokkos<double>;
```

Comment:
Is this file included in the CMakeLists.txt in the given dir as a standalone for the target?

Reply:
Yes, this is included as a standalone src file in the CMakeLists.txt.